server/control.py

```diff
@@ -20,18 +22,13 @@ def __init__(self, game_service: GameService, player_service: PlayerService):
         self.player_service = player_service
 
     def games(self, request):
-        body = repr(list(map(Game.to_dict, self.game_service.all_games))).encode()
-        return web.Response(body=msgpack.packb(body), content_type='application/x-msgpack')
+        body = dumps(list(map(lambda p: p.to_dict(), self.game_service.all_games))).encode()
+        #body = dumps(list(self.game_service.all_games).__dict__).encode()
+        return web.Response(body=body, content_type='application/json')
 
     def players(self, request):
-        body = list(map(lambda p: p.to_dict(), self.player_service.players.values()))
-        return web.Response(body=msgpack.packb(body), content_type='application/x-msgpack')
-
-    async def kick_player(self, request):
-        player = self.player_service.players[int(request.match_info.get('player_id'))]
-        assert isinstance(player.lobby_connection, LobbyConnection)
-        player.lobby_connection.kick()
-        return web.Response()
+        body = dumps(list(map(lambda p: p.to_dict(), self.player_service.players.values()))).encode()
```

Comment:
Maybe you can break out this part:
```python
list(map(lambda p: p.to_dict(), self.player_service.players.values()))
```

Into a reusable function. Something like:
```python
def to_dict_list(list_):
    return list(map(lambda p: p.to_dict(), list_))
```
Then this line becomes:
```suggestion
        body = dumps(to_dict_list(self.player_service.players.values()))
```
And you can also do it for the `/games` route as well.

And I would probably do the encode step separately for readability
```python
return web.Response(body=body.encode(), content_type='application/json')
```